api/bucketsd/service.go

```diff
@@ -3354,6 +3355,7 @@ func fromPbArchiveConfig(pbConfig *pb.ArchiveConfig) *mdb.ArchiveConfig {
 			MaxPrice:        pbConfig.MaxPrice,
```

Comment:
Some extra cascading to use the new `VerifiedDeal` field you added to:
- The struct that we keep in Hub saving the storage-config (mdb.ArchiveConfig)
- Map the `VerifiedDeal` to the storage-config being prepared to send to powergate when pushing the Cid (line 3387).